front-end/src/app/reports/transactions/transaction-type-picker/transaction-type-picker.component.ts

```diff
@@ -206,64 +214,99 @@ export class TransactionTypePickerComponent extends DestroyerComponent {
           ];
           break;
         case Disbursement.CONTRIBUTIONS_EXPENDITURES_TO_REGISTERED_FILERS:
-          transactionTypes = [
-            ScheduleBTransactionTypes.TRANSFER_TO_AFFILIATES,
-            ScheduleBTransactionTypes.CONTRIBUTION_TO_CANDIDATE,
-            ScheduleBTransactionTypes.CONTRIBUTION_TO_CANDIDATE_VOID,
-            ScheduleBTransactionTypes.CONTRIBUTION_TO_OTHER_COMMITTEE,
-            ScheduleBTransactionTypes.CONTRIBUTION_TO_OTHER_COMMITTEE_VOID,
-            ScheduleBTransactionTypes.IN_KIND_CONTRIBUTION_TO_CANDIDATE,
-            ScheduleBTransactionTypes.IN_KIND_CONTRIBUTION_TO_OTHER_COMMITTEE,
-            ScheduleFTransactionTypes.COORDINATED_PARTY_EXPENDITURE,
-            ScheduleFTransactionTypes.COORDINATED_PARTY_EXPENDITURE_VOID,
-            ScheduleETransactionTypes.INDEPENDENT_EXPENDITURE,
-            ScheduleETransactionTypes.INDEPENDENT_EXPENDITURE_VOID,
-            ScheduleETransactionTypes.MULTISTATE_INDEPENDENT_EXPENDITURE,
-            ScheduleETransactionTypes.INDEPENDENT_EXPENDITURE_CREDIT_CARD_PAYMENT,
-            ScheduleETransactionTypes.INDEPENDENT_EXPENDITURE_STAFF_REIMBURSEMENT,
-            ScheduleETransactionTypes.INDEPENDENT_EXPENDITURE_PAYMENT_TO_PAYROLL,
-          ];
+          if (isF3) {
```

Comment:
At this point would it make sense to define what transaction types are available for each form in reports.model and then filter the complete list of each group by whether or not they're current report type uses it?

Since we're here, I think this switch statement has maybe overstayed it's welcome 😆.  Looking at my original commit, i think it was asking too much of the humble switch anyways.  

I noticed you refactored the groups.  Would it make sense to take that to its logical conclusion and have the value of the groups be the actual list of transaction types in that group?  Then here we don't have to have any conditionals, we just reference the list of types defined by the group in question